docs/configurations.md

```diff
@@ -342,6 +342,54 @@ The export options can be set per model and includes the following options:
 - `export_max_rows`: Maximum number of rows to be exported. Default value is `0` which means unlimited.
 - `export_types`: List of export types to be enabled. Default value is `["csv","json"]`.
 
+## Pretty CSV Export
+- `ModelView.use_pretty_export`: Default value is `False`
+
+Enables exporting CSV files with user-friendly column labels and formatted cell values 
+matching the UI list view. 
+When enabled, exports utilize the `column_formatters` and `column_labels` defined in the admin view, 
+improving readability and ensuring consistency between the UI and exported data.  
+
+Custom cell formatting can be implemented in the ModelView class by overriding the async method 
+`custom_export_cell`.
```

Comment:
```suggestion
Custom cell formatting can be implemented in the ModelView class by overriding the async method 
`custom_export_cell`, otherwise basic cell formatting is used by default.
```